lib/util/notification_utils.dart

```diff
@@ -24,6 +26,33 @@ class _NotificationUtils {
       iOS: initializationSettingsIOS,
     );
     await localNotificationsPlugin.initialize(initializationSettings);
+
+    return _requestPermissions();
+  }
+
+  Future<bool> _requestPermissions() async {
+    if (kIsWeb) {
+      return true;
```

Comment:
what does this mean?

Reply:
![image](https://github.com/EnsembleUI/ensemble/assets/36638657/b54714b8-610a-462d-8357-4f4953dd3b2e)
I picked it from flutter local notification source code..
Basically flutter web has the local notification permission.